ipn/ipnlocal/network-lock.go

```diff
@@ -96,12 +96,92 @@ func (b *LocalBackend) tkaSyncIfNeededLocked(nm *netmap.NetworkMap) error {
 	}
 
 	if isEnabled && b.tka.authority.Head() != nm.TKAHead {
-		// TODO(tom): Implement sync
+		if err := b.tkaSyncLocked(ourNodeKey); err != nil {
+			return fmt.Errorf("tka sync: %v", err)
```

Comment:
```suggestion
			return fmt.Errorf("tka sync: %w", err)
```

This happens throughout the PR